WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostCardActionsHandler.kt

```diff
@@ -141,6 +145,17 @@ class ReaderPostCardActionsHandler @Inject constructor(
         }
     }
 
+    suspend fun handleReportPostClicked(post: ReaderPost) {
```

Comment:
We might want to consider re-using this code even in the legacy fragment. We re-use most (maybe even all) the other actions. Wdyt? If you decide to make this change, feel free to make it in develop for 15.8.

Reply:
@malinajirka - already have a note to update, because legacy functionality is different in 15.8. I'll drop a task on the RI board too! Thanks

Reply:
ohh, I see - I forgot about that. Thanks;)